backend/src/db/seed/truncate.ts

```diff
@@ -0,0 +1,16 @@
+import { Prisma, PrismaClient } from '@prisma/client';
+import { isProd } from 'common';
+
+export async function truncateAllTables(prisma: PrismaClient | Prisma.TransactionClient) {
+  if (isProd()) throw new Error('Please stop whatever you are doing right now.');
```

Comment:
not that we already have this function in testUtil.ts - it should just live in one file. we don't want to import testUtil.ts unless we're running tests because importing vitest stuff causes issues during normal operation, that's why i moved this function to its own file